README.md

```diff
@@ -123,7 +124,7 @@ Returns a resolved bluebird promise with the given value
 
 Returns a rejected bluebird promise with the given value.
 
-*Note: If the given value is a String, that string will be wrapped in an Error object and will be on the message property.*
+*Note: If the given value is a String, that string will be wrapped in an Error object and will be on the message property.* **Depreciated in >=2.0.0**
 
```

Comment:
"Deprecated" although that implies that the functionality is still there, but no longer recommended for use. In this case "Removed" would be more appropriate.